client/src/components/ProductDetail/galleryModalStyles.css

```diff
@@ -1,10 +1,3 @@
-button {
```

Comment:
hey can you please change the 'button' keyword to your className or ID so it doesnt affect all our buttons?